src/test/ui/macro_backtrace/main.rs

```diff
@@ -16,11 +16,14 @@
 
 // a local macro
 macro_rules! pong {
-    () => { syntax error }; //~ ERROR expected one of
+    () => { syntax error };
     //~^ ERROR expected one of
+    //~| ERROR expected one of
```

Comment:
... would it be better to move these annotations outside of the `macro_rules! pong { ... }`? I was a little confused at first by the main.stderr until I realized that they were just transcribed from the source because they happen to occur within the macro definition?

Reply:
I'm thinking something along the lines of:

```rust
// a local macro
macro_rules! pong {
    () => { syntax error };
}
//~^^ ERROR expected one of
//~| ERROR expected one of
//~| ERROR expected one of
```

but maybe I misremember the exact syntax here.

Its possible that even this may work:

```rust
// a local macro
macro_rules! pong {
    () => { syntax error };
} //~^ ERROR expected one of
  //~| ERROR expected one of
  //~| ERROR expected one of
```